### PR TITLE
kodi: update to Kodi 18 alpha 1

### PR DESCRIPTION
**Pull request by MilhouseVH**
Work in progress - don't merge.

Will update as and when necessary.

@peak3d do you have a Kodi18 version of your aml timing patch?

### CODE REVIEW

File: packages/mediacenter/kodi/package.mk [hunk 1]
```diff
@@ -32,7 +32,7 @@ PKG_LONGDESC="Kodi Media Center (which was formerly named Xbox Media Center or X
 PKG_IS_ADDON="no"
 PKG_AUTORECONF="no"
 
-PKG_CMAKE_SCRIPT="$ROOT/$PKG_BUILD/project/cmake/CMakeLists.txt"
+PKG_CMAKE_SCRIPT="$ROOT/$PKG_BUILD/CMakeLists.txt"
```
Comment (stefansaraev) [inline packages/mediacenter/kodi/package.mk]: PKG_CMAKE_SCRIPT is not needed now. $ROOT/$PKG_BUILD/CMakeLists.txt is the default

Comment (MilhouseVH) [inline packages/mediacenter/kodi/package.mk]: Thanks - I'll drop that when my broadband is back next week.

**Comment by lrusak** [comment]
How's this going?

**Comment by MilhouseVH** [comment]
It's fine, and it's what I use in my builds (I pull the latest add-ons separately). It's going to need add-on bumps as/when that happens, unless you think it should be merged before alpha2 (which may be a long while down the road).

**Pull request merged**